source/tests/system/nidaqmx_driver_api_tests.cpp

```diff
@@ -887,6 +887,24 @@ class NiDAQmxDriverApiTests : public Test {
     return status;
   }
 
+  ::grpc::Status perform_bridge_offset_nulling_cal_ex(PerformBridgeOffsetNullingCalExResponse& response)
```

Comment:
Your two new functions aren't referenced by anything. Can you add a TEST_F below for each similar to `self_cal` being referenced by the test `TEST_F(NiDAQmxDriverApiTests, SelfCal_Succeeds)` further below in this file.

Reply:
Note that unlike self-cal, strain cal requires a task. So the new tests also need to create a task, create a bridge/strain/thermocouple channel, and clear the task. It would make sense to add 2 or 3 more create channel helpers for strain, thermocouple, and possibly bridge.

Reply:
Thanks for pointing this out, I have added TEST_F for each of the functions and passed in a task, channel and clear the task after execution. However I am unable to test it for now because I'm facing build issues, I am just eyeballing it and will test it once I am able to.